### DISCUSSION TITLE
Project Status

### DISCUSSION THREAD

**Discussion by ivodolenc** [pngwn/MDsveX#678] [General]
Hi, sorry to ask, I don't mean to be rude, but is this project still active and is the mentioned migrations to `Svelte 5` being worked on?

I'm interested because I'm planning to build a completely new documentation site that would depend on `MDsveX` under the hood.

The website would be an advanced project with lots of docs content pages, components, etc., so I wouldn't want to be in a situation where drastic breaking changes happen in the middle of development.

I notice you already said that you don't have a firm date for the roadmap and that some basic issues like types, etc. haven't been addressed for months. I'm worried that you might have abandoned the project, which would be a real shame because this project is really cool and almost essential to the Svelte community for blog or docs sites.

**Comment by ivodolenc** [comment]
Hi, since there haven’t been any replies for months, I decided to create a new [`Svelte Markdown`](https://github.com/hypernym-studio/svelte-markdown) preprocessor.

I really need `Markdown` support for my new projects in SvelteKit, and I’m hoping it’ll be useful to others too.

`Svelte Markdown` is inspired by `MDsveX` — a big shoutout to its authors for their fantastic work!

I’ve made some slight API modifications and improvements, but the migration process should be pretty straightforward.

As it’s still in the early stages, I’d like to test it before releasing a stable version to avoid introducing major breaking changes.

The goal is to stay compatible with popular tools like **Remark** and **Rehype** for smooth content transformation when needed, while also ensuring robust support for `Svelte 5`, including built-in types and common unified plugins for easy imports.

The project is open-source, so ideas, suggestions, and code contributions are more than welcome. Feel free to check it out! Happy coding 🦾👨🏻‍💻

**Comment by pngwn** [comment]
Hail. Sorry. Yes this is still active and will get some more love very shortly. There isn't much stopping people from using this with v5 right now so there aren't any major blockers on that front. That said, there are a few bugs that desperately need adressing and will be my priority in the coming weeks.

**Comment by ivodolenc** [comment]
Thanks for the info, this is great news! 👍🏻

**Comment by lure23** [comment]
@ivodolenc wrote:
>almost essential to the Svelte community for blog or docs sites.

I agree.

A *lot* could be gained by simply addressing SvelteKit 2 in the [early sections of the docs](https://mdsvex.pngwn.io/docs#mdsvex-1). It now starts with: "*The `mdsvex` preprocessor function is a named import from the `mdsvex` module. Add it as a preprocessor to your rollup or webpack config, and tell the Svelte plugin or loader to also handle `.svx` files.*"

As you know, with SvelteKit one doesn't need to deal with either Rollup or Webpack. Reading this makes me think SvelteKit is not considered - though it likely is...

I found [this blog](https://joshcollinsworth.com/blog/build-static-sveltekit-markdown-blog) <sub>via [this comment](https://github.com/pngwn/MDsveX/issues/468#issuecomment-1220921104)</sub> that essentially Saved the Day and my site is up. The blog is not only about dynamic routes, but provides a full, step-by-step tutorial on how to merge SvelteKit and mdsvex together.

Related. I would offer myself to help with **cleanup** of the Issues. I think this plays a major role in the feel of abandonment one might get of this project. As a first step would be to make a sheet that groups issues by context/category and suggests whether they should be closed, updated, or kept as-is. We can then discuss the actions based on said sheet, and hopefully remove the set of Issues to only keep the relevant ones open.

There are related issues:

- https://github.com/pngwn/MDsveX/issues/311

**Reply by einarpersson** [reply]
I fully agree with this. Please take this @pngwn as a sign of appreciation and consider accepting this help being offered with cleaning up issues and obsolete docs. It would probably go a long way!
